docs/patterns.json

```diff
@@ -12040,7 +12058,7 @@
         },
         {
           "name": "langDir",
-          "default": "C:\\Users\\Vítor Pires\\repos\\codacy\\codacy-eslint\\node_modules\\eslint-plugin-spellcheck\\rules/utils/dicts"
+          "default": "/Users/vrhpires/dev/codacy-eslint/node_modules/eslint-plugin-spellcheck/rules/utils/dicts"
```

Comment:
I think this should not be like this, right? We should fix this, as this is pointing to a directory in a local machine.

Reply:
It would be desirable not to happen. I discussed this with Betul and within the 2nd line support team.

It comes from here:
https://github.com/aotaduy/eslint-plugin-spellcheck/blob/a114c3454a5120c10d7e38d28401b26e62655c68/rules/defaultSettings.js#LL2C14-L2C24

There is nothing we can do other than add the patterns.json to the github ignore file.
Because we have to **generateDocs** and it runs automatically the **defaultSettings.js** from that package. It then uses **__dirname** which gets the absolute path from the host and writes the output to **patterns.json**.

It has always been like this and we've decided not to pursue an improvement.